.github/workflows/release.yaml

```diff
@@ -39,6 +46,9 @@ jobs:
           gsutil -m cp -r gs://$GCS_BUCKET/metal-os/stable/firewall/3.0-ubuntu gs://$GCS_BUCKET/metal-os/${GITHUB_REF##*/}/firewall/3.0-ubuntu
           gsutil -m cp -r gs://$GCS_BUCKET/metal-os/stable/ubuntu/24.04 gs://$GCS_BUCKET/metal-os/${GITHUB_REF##*/}/ubuntu/24.04
 
+          docker push ghcr.io/metal-stack/debian:12.${GITHUB_REF##*/}
+          docker push ghcr.io/metal-stack/ubuntu:24.04.${GITHUB_REF##*/}
```

Comment:
This must be done for all images + this will not work in the current stage, I guess. The images first need to be pulled from stable / latest.